sw/device/tests/alert_handler_reverse_ping_in_deep_sleep_test.c

```diff
@@ -175,7 +204,18 @@ static void check_local_alerts(void) {
  * line to the CPU, which results in a call to this OTTF ISR. This ISR
  * overrides the default OTTF implementation.
  */
-void ottf_external_isr(void) { interrupt_serviced = true; }
+void ottf_external_isr(void) {
+  // Specifically assert that no alerts were triggered
+  check_local_alerts();
+  dif_rv_plic_irq_id_t irq_id;
+  CHECK_DIF_OK(dif_rv_plic_irq_claim(&plic, kPlicTarget, &irq_id));
+  LOG_INFO("Saw interrupt %d", irq_id);
+
+  // Record an interrupt has been serviced so the test can fail if unexpected.
+  interrupt_serviced = true;
+
+  log_external_alerts();
```

Comment:
Printing out all the info inside the ISR will take a lot of time, which in general is not good practice in embedded systems, where we want to keep the system responsive. Since this is just a test that doesn't enable many interrupts its probably OK, but perhaps next time you would consider having:

```c
if (interrupt_serviced) {
    log_external_alerts();
}
``` 
after the corresponding `wait_for_interrupt` below, or just first thing in the `main()`.

Reply:
Doing so would also make this similar to how the `check_local_alerts` is executed.